certdocs/attributes.json

```diff
@@ -1,16 +1,30 @@
 {
-	"attributes": [{
-			"name": "Parents"
-		},
-		{
-			"name": "Rationale"
-		},
-    {
-			"name": "Verification",
-			"value": "(Demonstration|Unit [Tt]est|[Tt]est)"
-		},
-		{
-			"name": "Safety Impact"
-		}
-	]
+    "attributes": [
+        {
+            "name": "Parents",
+            "filter": "^REQ-TRAQ-SWH",
+            "required": "any",
+            "value": "^(?:REQ-TRAQ-SYS-\\d+(?:,|, )?)*$"
+        },
+        {
+            "name": "Parents",
+            "filter": "^REQ-TRAQ-SWL",
+            "required": "any",
+            "value": "^(?:REQ-TRAQ-SWH-\\d+(?:,|, )?)*$"
+        },
+        {
+            "name": "Rationale",
+            "filter": "^REQ",
+            "required": "any"
+        },
+        {
+            "name": "Verification",
+            "filter": "^REQ",
+            "value": "(Demonstration|Unit [Tt]est|[Tt]est)"
+        },
```

Comment:
(assume you mean Requirements Standards)

that's a very good point, I agree, i think it would  be better to specify a verification method at each level, although the allowed options would be different (for example, you could "flight test" a system level requirement, but clearly that wouldn't make sense for a low-level requirement).

I will leave the reqtraq attributes as it is for now, but when we come to create a schema for the full system will bear this in mind, plus update out requirements standards.

Reply:
Yes, I meant Requirement Standards, I mixed them up. I will keep this in mind for the reqtraq config across multiple repos too.